src/main/java/net/dv8tion/jda/core/managers/impl/AudioManagerImpl.java

```diff
@@ -65,8 +66,8 @@
 
     public AudioManagerImpl(Guild guild)
     {
-        this.guild = guild;
-        this.api = (JDAImpl) guild.getJDA();
+        this.guild = (GuildImpl) guild;
```

Comment:
JDA doesn't provide the user with an impl. It only gives them the interface which doesn't define setters. Unless the user makes a cast to the impl class they cannot access internal methods.